### ISSUE TITLE
warning: refname 'packages/packages/pacman' is ambiguous.

### ISSUE THREAD

**Issue by NicoHood** [falconindy/asp#18]
```
[arch@arch asp]$ asp list-arches linux
x86_64
i686
[arch@arch asp]$ asp list-arches bash
x86_64
i686
[arch@arch asp]$ asp list-arches pacman
warning: refname 'packages/packages/pacman' is ambiguous.
x86_64
i686
```

**Comment by falconindy** [comment]
This is already fixed at head, see 8d504c6b.

**Comment by NicoHood** [comment]
Any chance to get bugfix releases aka 1.0.1 for those fixes?
